utility/cli_message_examples_reprex.md

```diff
@@ -147,8 +202,8 @@ missing_col_supertbl <- tibble(redcap_data = list()) %>%
   as_supertbl()
 make_labelled(missing_col_supertbl)
 #> Error in `make_labelled()`:
-#> ✖ You've supplied `<redcap_supertbl[,1]>` for `supertbl` which is not a
-#>   valid value
+#> ✖ You've supplied `<rdcp_spr[,1]>` for `supertbl` which is not a valid
```

Comment:
Why is this `rdcp_spr?` and not `redcap_supertbl`?

Reply:
Huh, strange. It looks like it has to do with `rlang::as_label()` under our formatter function, wondering if there was an update since the last time we ran this `reprex`.

https://github.com/CHOP-CGTInformatics/REDCapTidieR/blob/fe147b4e476c496f5ded1dfc54b4a66601ce53fc/R/checks.R#L531

@ezraporter maybe we can look over tomorrow.

Reply:
We think this came up in one of the `rlang` updates (since the original run of this file in December 2022, we went from v1.0.6 to v1.1.1), but it was tough to diagnose since we now have dependencies in the package that require these later versions.

What we did find was that `as_label()` in `rlang` makes use of [`vctrs::vec_ptype_full()`](https://vctrs.r-lib.org/reference/vec_ptype_full.html) which takes the first element of a class vector ("redcap_supertbl") and by default abbreviates it to 8 characters. 

For now I think we should leave as is for this release, but we can open an issue if it's something we want finer control over to address later.

Reply:
Resolving for now, opened a new issue here: https://github.com/CHOP-CGTInformatics/REDCapTidieR/issues/157